cache/edge_lambdas/src/redirector.ts

```diff
@@ -80,6 +80,13 @@ export const getRedirect = (
         ? 'www-e2e.wellcomecollection.org'
         : 'wellcomecollection.org';
 
+  if (
+    uriSansSlash.startsWith('/articles') &&
+    literalRedirects[uriSansSlash] === undefined
+  ) {
+    return redirect301(host, uriSansSlash.replace(/^\/articles/, '/stories'));
+  }
+
```

Comment:
I don't have a real suggestion for this, but I think we might want to keep specific redirections out of this file (keeping them centralised) as everything else here is very generic.
Maybe a new section in `redirects.ts` like `queryRedirects` but made simpler? For straightforward higher level redirections? What do you think? (Might be useful for other route changes moving forward too).

Reply:
good point, do you think the change is better?

Reply:
Yes exactly something like that! I feel like it'll definitely be coming in handy. 

A few questions on there that I don't have the answers to BUT what I realised yesterday is that the redirects apply to the e2e environment. So if we make it so it doesn't scale it down, we could test the redirects from the e2e build on this PR. (Could be done by either failing the e2e or commenting out the task that scales it down).

Reply:
As per our chat, ignore me 😄